tests/Core/RandomExtensionTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using FsCheck.Xunit;
+using Xunit;
+
+namespace Bearded.Utilities.Tests.Core
+{
+    public class RandomExtensionTests
+    {
+        [Property]
+        public void NextLong(int seed)
+        {
+            var random = new Random(seed);
+
+            random.NextLong();
+        }
+
+        [Property]
+        public void NextLongWithMax(int seed, long max)
+        {
+            var random = new Random(seed);
+
+            var number = random.NextLong(max);
+            
+            Assert.InRange(number, 0, max - 1);
+        }
+
+        [Property]
+        public void NextLongWithMinAndMax(int seed, long min, long max)
```

Comment:
What happens if max < min? I find it a bit confusing to see a property test testing min and max without any restrictions on the generated min/max. Should we create a property generator for (min, max) interval pairs?